compiler/Makefile

```diff
@@ -3,10 +3,14 @@ COMPILER_JAR=../build/bin/nballerina.jar
 BAL?=bal
 BAL_BUILD_FLAGS ?= --offline
 
+ifneq (BAL_ALL_TESTS, true)
+BAL_TEST_FLAGS ?= --disable-groups longRunning
```

Comment:
I often do `make test` in the `compiler` directory. So I suggest in `compiler/Makefile` do

```
BAL_TEST_FLAGS ?= $(BAL_BUILD_FLAGS) --disable-groups longRunning

test:
        $(BAL) test $(BAL_TEST_FLAGS)
```

Then people can run all tests with `bal test`.